injects/logger.js

```diff
@@ -32,7 +56,17 @@ const out = encoder.pipe(
 // log stack traces, export a flag to opt out of logging for internals
 exports.skipThis = false
 const skipAsyncIds = new Set()
+let firedOnce = false;
+
 const hook = asyncHooks.createHook({
+  before () {
+    if (!firedOnce && process.mainModule && checkForTranspiledCode(process.mainModule.filename)) {
+      // Show warning to user
+      fs.writeSync(3, 'source_warning', null, 'utf8')
+      firedOnce = true
+    }
+  },
```

Comment:
Please don't add a before hook. This is going to slow down things even more :/.

Reply:
No problem. Is there a place I can add it where process.mainModule would be available?

Reply:
@mcollina - We will be resolving this by introducing a process.nextTick and putting the check in there.